### ISSUE TITLE
Going full screen in Firefox 42 is horrible

### ISSUE THREAD

**Issue by tjwood100** [adamzap/landslide#184]
Going full screen in Firefox 42 results in a very jagged resize. The reason is that the browser calls several onresize events and that toggles back and forth between expanded and non-expanded view.

I added a function called fitSlides very similar to expandSlides but without toggling:

```
var fitSlides = function() { // TJW
    if (overviewActive) {
        return;
    }
    scale = 1;
    if(expanded)
    {
        scale = computeScale();
    }
    setScale(scale);
};
```

and in the initialize function changed window.onresize to this function:

```
    window.onresize = fitSlides;
```

...and that seems much nicer.

**Comment by adamzap** [comment]
I'm having trouble reproducing this. Which operating system are you on?

**Comment by tjwood100** [comment]
I was using Ubuntu 14.04 (default Unity interface) and saw the issue with both Firefox and Chromium (latest packaged versions). 

I will check later if I can reproduce it on Windows.

**Comment by tjwood100** [comment]
Actually I was mistaken. It only seems to affect Firefox. Using version 42 I can reproduce on both Ubuntu 14.04 and Windows 7.  It seems like the exact effect can vary a little I guess depending on how many times onresize ends up getting called:

Videos showing the effect:

http://files.tx9.co.uk/landslide-ubuntu.ogv (1.3 MB)
http://files.tx9.co.uk/landslide-windows.ogv (1.1 MB)

**Comment by adamzap** [comment]
Looks good to me in Firefox v66.0.3.

I guess they fixed the issue sometime in the last 3.5 years 😉
